systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -289,51 +290,62 @@ def post_configure(self, address):
         route = CsRoute()
         if not self.get_type() in ["control"]:
             route.add_table(self.dev)
-            
             CsRule(self.dev).addMark()
+
             self.check_is_up()
-            self.set_mark()
+
+            if self.dnum != '0':
```

Comment:
self.dnum is hex, so this line always return true.
you can use ( self.dev != 'eth0') instead, or other